cmd/utils/flags.go

```diff
@@ -507,6 +507,21 @@ var (
 		Usage: "HTTP path path prefix on which JSON-RPC is served. Use '/' to serve on all paths.",
 		Value: "",
 	}
+	HTTPRequestReadTimeout = cli.IntFlag{
+		Name:  "http.timeout.read",
+		Usage: "Timeout for HTTP-RPC read requests",
+		Value: node.DefaultHTTPTimeoutRead,
```

Comment:
Instead of using this and duplicate the value for that new node variable, why don't you use instead:
`rpc.DefaultHTTPTimeouts.ReadTimeout`
and include the rpc import?
That ReadTimeout is already multiplied by the seconds, but we can divide it here by `time.Second`
This way, if we change the defaults (for some reason) in the future, everything will be correct